plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -147,3 +148,41 @@ wizard.org.jkiss.dbeaver.core.wizard.import.scripts.description = SQL \u811A\u67
 wizard.org.jkiss.dbeaver.core.wizard.import.scripts.name        = \u811A\u672C
 wizard.org.jkiss.dbeaver.core.wizard.new.connection.description = \u6570\u636E\u5E93\u8FDE\u63A5
 wizard.org.jkiss.dbeaver.core.wizard.new.connection.name        = \u6570\u636E\u5E93\u8FDE\u63A5
+extension-point.org.jkiss.dbeaver.clearHistoryHandler.name = \u6E05\u9664\u5386\u53F2\u7BA1\u7406\u5668
+command.org.jkiss.dbeaver.core.commit.label = \u63D0\u4EA4
+command.org.eclipse.ui.navigate.expandAll.name = \u5C55\u5F00\u5168\u90E8
+command.org.eclipse.ui.navigate.collapseAll.name = \u9690\u85CF\u5168\u90E8
+command.org.eclipse.ui.edit.paste.name = \u7C98\u8D34
+command.org.eclipse.ui.edit.copy.name = \u590D\u5236
```

Comment:
In general, Eclipse commands should not be localized - it's Babel's job to localize those. Moreover, these keys are not used and don't affect the actual product.